libnmstate/nm/connection.py

```diff
@@ -206,6 +208,11 @@ def create_new_nm_simple_conn(iface, nm_profile):
             create_macvlan_setting(iface_info, nm_profile, tap=True)
         )
 
+    if Veth.CONFIG_SUBTREE in iface_info:
+        veth_setting = create_veth_setting(iface_info, nm_profile)
```

Comment:
By passing `VethIface`, you can save `nm/veth` from parsing dictionary. Let's say the `create_veth_setting()` can just use `veth_setting.props.peer = iface.peer`

Reply:
Done! Thanks!